Fix: Open element in viewer mode bug fix (#521)
finos/legend-studio: .changeset/curvy-seahorses-rest.md

---
'@finos/legend-studio': patch
---

> Bug fix for opening element in viewer mode - View in Project

Reviewer: ```suggestion
Fix an issue with opening a particular element in viewer mode from Studio editor not working properly (#[https://github.com/finos/legend-studio/issues/503](https://github.com/finos/legend-studio/issues/503))
```

> Fix an issue with opening a particular element in viewer mode from Studio editor ([#503](https://github.com/finos/legend-studio/issues/503))

Reviewer: ```suggestion
Fix an issue with opening a particular element in viewer mode from main editor ([#503](https://github.com/finos/legend-studio/issues/503))
```

> Fix an issue with opening a particular element in viewer mode from Studio editor (#[https://github.com/finos/legend-studio/issues/503](https://github.com/finos/legend-studio/issues/503))

Reviewer: ```suggestion
Fix an issue with opening a particular element in viewer mode from Studio editor ([#503](https://github.com/finos/legend-studio/issues/503))
```

> Fix an issue with opening a particular element in viewer mode from Studio editor not working properly (#[https://github.com/finos/legend-studio/issues/503](https://github.com/finos/legend-studio/issues/503))

Reviewer: ```suggestion
Fix an issue with opening a particular element in viewer mode from Studio editor (#[https://github.com/finos/legend-studio/issues/503](https://github.com/finos/legend-studio/issues/503))
```
